rust/operator-binary/src/config.rs

```diff
@@ -67,12 +67,12 @@ fn append_ldap_config(config: &mut BTreeMap<String, String>, ldap: &LdapAuthenti
         AirflowConfigOptions::AuthLdapServer.to_string(),
         format!(
             "{protocol}{server_hostname}:{server_port}",
```

Comment:
Nit: It would be cool to have something like https://github.com/stackabletech/operator-rs/blob/f02577a2651e37e0b9627cf7c44c816029ad0c4c/src/commons/authentication/oidc.rs#L113 for LDAP as well ;)

Reply:
Should this be a separate PR? It would need some work in operator-rs.

Happy to do it now though, to save another round of changes for every operator.

Reply:
Awesome! I would be happy to release a new op-rs version and update this PR to bump to the new version

Reply:
Cool, almost done with that change

Reply:
Here's the PR: https://github.com/stackabletech/operator-rs/pull/705